### PR TITLE
fix wt2 attributes

### PR DESCRIPTION
**Pull request by pb82**
Fixes the attributes for WT2

### CODE REVIEW

File: public/walkthroughs/walkthrough-2/walkthrough.adoc [hunk 1]
```diff
@@ -15,6 +15,7 @@
 :3Scale-ProductVersion: 2.3
 :EnMasse-master-ProductLongName: EnMasse
 :walkthrough: Integrating API-driven applications
+:fuse-flights-aggregator-app-name: fuse-flights-aggregator
```
Comment (pb82) [inline public/walkthroughs/walkthrough-2/walkthrough.adoc]: @aidenkeating i logged all the available attributes but there wasn't one for the aggregator app name. Should there be one?

Comment (aidenkeating) [reply public/walkthroughs/walkthrough-2/walkthrough.adoc]: @pb82 There wouldn't be one for the app name as that's walkthrough specific. There should be one for the route though

Comment (pb82) [reply public/walkthroughs/walkthrough-2/walkthrough.adoc]: Yes, `route-fuse-flights-aggregator-host` works for the route

**Comment by aidenkeating** [comment]
:eyes:

**Comment by pb82** [comment]
@aidenkeating @mfrances17 thanks for reviewing. I'll give it another speedrun and then merge.

**Pull request merged**